matRad_calcGeoDists.m

```diff
@@ -17,8 +17,8 @@
 %                                  lateralCutOff)
```

Comment:
Explaination of radDepthIx missing in function call description and for description of function arguments below.